frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -68,7 +76,39 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
             },
             deleteFlag: (state, { id }) => state.filter((flag) => flag.id !== id),
         },
+        activeTab: [
+            FeatureFlagsTabs.OVERVIEW,
+            {
+                setActiveTab: (state, { tabKey }) =>
+                    Object.values<string>(FeatureFlagsTabs).includes(tabKey) ? tabKey : state,
+            },
+        ],
+        historyPage: [null as number | null, { setHistoryPage: (_, { page }) => page }],
     },
+    actionToUrl: ({ values }) => ({
+        setActiveTab: () => {
+            const searchParams = {
+                ...router.values.searchParams,
+                tab: values.activeTab,
+            }
+            if (values.activeTab !== FeatureFlagsTabs.HISTORY) {
+                delete searchParams['page']
+            }
+            return [router.values.location.pathname, searchParams, router.values.hashParams, { replace: true }]
+        },
```

Comment:
Search params are kind of ugly. Here's a proposal for a slightly more elegant structure (that's still backwards-compatible):
- overview is `/feature_flags`
- history is `/feature_flags/history`

Reply:
hmmm, `/feature_flags/history` was matched by the feature flag route `featureFlag: (id: string | number) => `/feature_flags/${id}`,`

We've got multiple pages that track tabs as search params like this so I'd suggest we stick with it. 

Plus, most users don't look at the URL tbh

E.g. in safari you are shown this:

<img width="821" alt="Screenshot 2022-03-24 at 18 50 32" src="https://user-images.githubusercontent.com/984817/159989338-b23b0269-fda9-42f0-aa8d-800847d23eae.png">